docs/package.json

```diff
@@ -12,17 +12,18 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"scripts": {
-		"build": "concurrently npm:build:api-documentation npm:build:md-magic && npm run hugo",
+		"build": "concurrently npm:build:api-documentation npm:build:md-magic npm:build:redirects && npm run hugo",
```

Comment:
There are still some unneeded overlaps between a number of the scripts here. I'm going to clean them up, though I'll do it in a separate PR.